### ISSUE TITLE
$cs-hue-offset should be in degrees

### ISSUE THREAD

**Issue by ducdebreme** [north/generator-style-prototype#40]
According to https://github.com/Team-Sass/color-schemer/blob/master/stylesheets/color-schemer/_color-schemer.scss $cs-hue-offset is used in adjust-hue(), which accepts degrees as 2nd parameter (refer to http://sass-lang.com/documentation/Sass/Script/Functions.html#adjust_hue-instance_method)

I have added a patch

```
diff --git a/app/templates/sass/global/variables/_colors.scss b/app/templates/sass/global/variables/_colors.scss
index 95bd5ae..a4e6b29 100644
--- a/app/templates/sass/global/variables/_colors.scss
+++ b/app/templates/sass/global/variables/_colors.scss
@@ -15,8 +15,8 @@ $cs-primary: #fdb252;
 // Primary color, can be any color you can use in CSS, plus CMYK from Color Schemer
 $cs-scheme:             accented-analogic;
 // Options: mono, complement, triad, tetrad, analogic, accented-analogic
-$cs-hue-offset:         42;
-// Options: 0-100;
+$cs-hue-offset:         45deg;
+// Options: 0-90deg;
 $cs-brightness-offset:  10;
 // Options: 0-100;
 $cs-color-model:        ryb;
```

**Comment by Snugug** [comment]
If you wish to have a change considered, please submit it as a pull request, not a patch.

**Comment by ducdebreme** [comment]
done > see https://github.com/Team-Sass/generator-style-prototype/pull/41

**Comment by Snugug** [comment]
This is no longer included in 2.0
